podman_compose.py

```diff
@@ -2229,9 +2229,9 @@ def _parse_compose_file(self) -> None:
 
         self.x_podman = compose.get("x-podman", {})
         self.x_podman.update({
-            key.removeprefix("PODMAN_COMPOSE_").lower(): value  # type: ignore[misc]
+            key.removeprefix("PODMANCOMPOSE_").lower(): value  # type: ignore[misc]
```

Comment:
Better use `PODMAN_COMPOSE` prefix. Not merging names is common across podman ecosystem, e.g. `PODMAN_COMPOSE_PROVIDER` and other variables affecting how podman works.